modules/lb/lexicon/allophones.lb.xml

```diff
@@ -1,136 +1,139 @@
-    <vowel ph="9" vlng="s" vheight="2" vfront="2" vrnd="+"/>
-    <vowel ph="U" vlng="s" vheight="1" vfront="3" vrnd="+"/>
-    <vowel ph="O" vlng="s" vheight="2" vfront="3" vrnd="+"/>
-    <vowel ph="a" vlng="s" vheight="3" vfront="2" vrnd="-"/>
-
-    <vowel ph="aI" vlng="d" vheight="3" vfront="1" vrnd="-"/>
-    <vowel ph="OY" vlng="d" vheight="2" vfront="3" vrnd="+"/>
-    <vowel ph="aU" vlng="d" vheight="3" vfront="2" vrnd="+"/>
+    <vowel ph="y:" ipa="yː" vlng="l" vheight="1" vfront="2" vrnd="+"/>
+    <vowel ph="y" ipa="y" vlng="s" vheight="1" vfront="2" vrnd="+"/>
+    <vowel ph="2:" ipa="øː" vlng="l" vheight="2" vfront="2" vrnd="+"/>
+    <vowel ph="2" ipa="ø" vlng="s" vheight="2" vfront="2" vrnd="+"/>
+
+    <vowel ph="I" ipa="ɪ" vlng="s" vheight="1" vfront="1" vrnd="-"/>
+    <vowel ph="Y" ipa="ʏ" vlng="s" vheight="1" vfront="2" vrnd="+"/>
+    <vowel ph="E" ipa="ɛ" vlng="s" vheight="2" vfront="1" vrnd="-"/>
+    <vowel ph="9" ipa="œ" vlng="s" vheight="2" vfront="2" vrnd="+"/>
+    <vowel ph="U" ipa="ʊ" vlng="s" vheight="1" vfront="3" vrnd="+"/>
+    <vowel ph="O" ipa="ɔ" vlng="s" vheight="2" vfront="3" vrnd="+"/>
+    <vowel ph="a" ipa="a" vlng="s" vheight="3" vfront="2" vrnd="-"/>
+
+    <vowel ph="aI" ipa="aɪ̯" vlng="d" vheight="3" vfront="1" vrnd="-"/>
+    <vowel ph="OY" ipa="ɔʏ̯" vlng="d" vheight="2" vfront="3" vrnd="+"/>
+    <vowel ph="aU" ipa="aʊ̯" vlng="d" vheight="3" vfront="2" vrnd="+"/>
 
     <!-- additional vowels imported from French -->
-    <vowel ph="3" vlng="l" vheight="1" vfront="1" vrnd="-"/>
+    <vowel ph="3" ipa="ɜ" vlng="l" vheight="1" vfront="1" vrnd="-"/>
 
-    <vowel ph="a~:" vlng="l" vheight="3" vfront="2" vrnd="-" ctype="n"/>
-    <vowel ph="E~:" vlng="l" vheight="2" vfront="1" vrnd="-" ctype="n"/>
-    <vowel ph="O~:" vlng="l" vheight="2" vfront="3" vrnd="+" ctype="n"/>
-    <vowel ph="o~:" vlng="l" vheight="2" vfront="3" vrnd="+" ctype="n"/>
-    <vowel ph="9~:" vlng="l" vheight="2" vfront="2" vrnd="+" ctype="n"/>
+    <vowel ph="a~:" ipa="ãː" vlng="l" vheight="3" vfront="2" vrnd="-" ctype="n"/>
+    <vowel ph="E~:" ipa="ɛ̃ː" vlng="l" vheight="2" vfront="1" vrnd="-" ctype="n"/>
+    <vowel ph="O~:" ipa="ɔ̃ː" vlng="l" vheight="2" vfront="3" vrnd="+" ctype="n"/>
+    <vowel ph="o~:" ipa="õː" vlng="l" vheight="2" vfront="3" vrnd="+" ctype="n"/>
+    <vowel ph="9~:" ipa="œ̃ː" vlng="l" vheight="2" vfront="2" vrnd="+" ctype="n"/>
+    <vowel ph="A~:" ipa="ɑ̃ː" vlng="l" vheight="3" vfront="3" vrnd="-" ctype="n"/>
 
     <!-- and their versions in French MaryTTS -->
-    <vowel ph="e~" vlng="l" vheight="2" vfront="1" vrnd="-" ctype="n"/>
-    <vowel ph="a~" vlng="l" vheight="3" vfront="2" vrnd="-" ctype="n"/>
-    <vowel ph="o~" vlng="l" vheight="2" vfront="3" vrnd="+" ctype="n"/>
-    <vowel ph="9~" vlng="l" vheight="2" vfront="2" vrnd="+" ctype="n"/>
-    <vowel ph="0" vlng="s" vheight="2" vfront="3" vrnd="+"/>
+    <vowel ph="e~" ipa="ẽ" vlng="l" vheight="2" vfront="1" vrnd="-" ctype="n"/>
```

Comment:
The `vlng="l"` was surprising given that there’s no colon in the SAMPA transcription, but I’ve left it unchanged.